client/lib/analytics/ad-tracking/setup.js

```diff
@@ -53,7 +53,9 @@ export function setup() {
 		// Linkedin
 		if ( mayWeInitTracker( 'linkedin' ) ) {
 			setupLinkedinInsight(
-				isJetpackCloud() || isJetpackCheckout() ? TRACKING_IDS.jetpackLinkedinId : null
+				isJetpackCloud() || isJetpackCheckout()
+					? TRACKING_IDS.jetpackLinkedinId
```

Comment:
This also means that we will track page view (when this condition is true), the Linkedin "base" script also gets loaded here https://github.com/Automattic/wp-calypso/blob/052d90fe7c5c9e39237daedbf917955f481ce69a/client/lib/analytics/ad-tracking/load-tracking-scripts.js#L94